EU4toV2/Source/EU4World/Country/EU4Country.cpp

```diff
@@ -607,6 +607,12 @@ bool EU4::Country::hasFlag(const std::string& flag) const
 	return itr != flags.end();
 }
 
+bool EU4::Country::hasReform(const std::string& reform) const
+{
+	const auto& itr = governmentReforms.find(reform);
+	return itr != governmentReforms.end();
```

Comment:
You can consolidate these two lines into a single statement. Better yet, use .count() instead of finding the actual reform.